packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -99,6 +130,20 @@ class CozyStackClient {
    * @throws {FetchError}
    */
   async fetchJSON(method, path, body, options = {}) {
+    try {
+      return await this.fetchJSON(method, path, body, options)
+    } catch (e) {
+      if (/Expired token/.test(e.message)) {
+        const token = await this.refreshToken()
+        this.setToken(token)
+        return await this.fetchJSON(method, path, body, options)
```

Comment:
Je ne sais pas si tu veux faire un `this.fetchJSON ` ou un `this._fetchJSON ` mais on peut rentrer dans une boucle infini non ?

Reply:
oui, c'est une erreur d'inattention, merci